README.md

```diff
@@ -93,6 +93,91 @@ A simple remote procedure call tool for Janet.
 (:close c)
 ```
 
+## Path
+
+Simple path manipulation module for Janet. Supports manipulation both
+windows and posix paths on any platform, and provides functions that
+work according to the current host platform.
+
+All functions have three forms, under `path`, `path/win`, and `path/posix`. The
+prefix
+indicates which type of path the function manipulates.
+
+```clojure
+(import spork/path)
+
+# Examples for a non-windows system, use path/win/ for windows and
```

Comment:
May be using backticks are `path/win/` is good?

Reply:
They are in the code already.

Reply:
Missed that -- sorry :)